operator/pkg/tasks/init/karmadaresource.go

```diff
@@ -200,8 +200,8 @@ func runAPIService(r workflow.RunData) error {
 		return fmt.Errorf("failed to apply aggregated APIService resource to karmada controlplane, err: %w", err)
 	}
 
-	waiter := apiclient.NewKarmadaWaiter(config, nil, time.Second*20)
-	if err := apiclient.TryRunCommand(waiter.WaitForAPIService, 3); err != nil {
+	waiter := apiclient.NewKarmadaWaiter(config, nil, time.Second*30)
+	if err := waiter.WaitForAPIService(constants.APIServiceName); err != nil {
```

Comment:
Before, the time was 20 seconds multiplied by 3, which equals 60 seconds. Now, it is only 30 seconds. Is this really enough time? It seems insufficient considering that pulling the image takes a long time.

Reply:
Make sence, I'm with you.
 In consideration of time to pull the image, it's more appropriate to set 60 seconds for this.